src/main/resources/i18n/messages_en.properties

```diff
@@ -100,3 +100,15 @@ email.notification.aux.plagiarismVerdict.plagiarism=Your case is considered as p
 email.notification.aux.plagiarismVerdict.point.deduction=Due to your plagiarism, we are deducting points from your score in this exercise!
 email.notification.aux.plagiarismVerdict.warning=Due to the plagiarism attempt, you are warned. Any further plagiarism can lead to severe consequences!
 email.notification.aux.plagiarismVerdict.no.plagiarism=The case is no longer considered as plagiarism.
+
+# Tutorial Groups
+email.notification.title.tutorialGroup.registrationStudent=You have been registered to the tutorial group {0} in the course {1} by {2}.
```

Comment:
In my opinion active would sound a little better, e.g., “{2} registered you to the tutorial group {0} in the course {1}“.

Reply:
Changed to active form in a2d98006db19207c790b367b9e6479bd55ba6c9d